webapp/app/js/perf.js

```diff
@@ -238,6 +238,17 @@ perf.controller('PerfCtrl', [ '$state', '$stateParams', '$scope', '$rootScope',
       $scope.reload();
     };
 
+    $scope.hideSeries = function(signature){
+      var plotdata = $scope.plot.getData();
+      for (var i = 0; i < plotdata.length; i++) {
```

Comment:
You could replace this iteration logic with an underscore function like:

var datumToHide = _.find(plotdata, function(datum) { return datum.thSeries.signature == signature });
datumToHide.points.show = !datumToHide.points.show;